src/IssueViz.Security/Taint/TaintList/ViewModels/TaintIssuesControlViewModel.cs

```diff
@@ -56,7 +56,10 @@ internal sealed class TaintIssuesControlViewModel : ITaintIssuesControlViewModel
         private readonly object Lock = new object();
         private string activeDocumentFilePath;
 
-        public ObservableCollection<ITaintIssueViewModel> Issues { get; } = new ObservableCollection<ITaintIssueViewModel>();
+        // TODO: duncanp - make collection private
+        internal /* for testing */ ObservableCollection<ITaintIssueViewModel> Issues { get; } = new ObservableCollection<ITaintIssueViewModel>();
```

Comment:
If we decide to keep this change I'll change the scope in a separate PR (changing it here would have added quite a bit of noise).